### PR TITLE
Add biscuit-related tools

### PR DESCRIPTION
**Pull request by njspix**
Adding two envs to facilitate alignment with Biscuit and the BiscuitBlaster one-liner (https://huishenlab.github.io/biscuit/biscuitblaster/)

### CODE REVIEW

File: combinations/hash.tsv [hunk 1]
```diff
@@ -195,3 +195,5 @@ bioconductor-diffbind=3.4.0,bioconductor-csaw=1.28.0,bioconductor-edger=3.36.0,r
 r-shazam=1.1.0,r-tigger=1.0.0
 bamcmp=2.2	quay.io/bioconda/base-glibc-debian-bash:latest	0
 ascat=3.0.0,cancerit-allelecount=4.3.0
+biscuit=1.0.2.20220113,samtools=1.14
```
Comment (bgruening) [inline combinations/hash.tsv:198]: why is this container needed if you have the samblaster one as well?

Comment (njspix) [reply combinations/hash.tsv:198]: It's not strictly required. However, I anticipate that some folks will not prefer the samblaster pipeline; I was hoping to minimize requirements for that case.

Comment (njspix) [reply combinations/hash.tsv:198]: Happy to just keep the 3-tool container if you think that's best

Comment (bgruening) [reply combinations/hash.tsv:198]: It would save storage and cost at our side.

Comment (njspix) [reply combinations/hash.tsv]: Sounds good. Removed the redundant line

**Comment by njspix** [comment]
Sounds good! Removed the redundant line

**Review by bgruening — APPROVED** [review]

**Comment by njspix** [comment]
I can’t find it with mulled-search and I see this on GitHub:

[Shape  Description automatically generated with medium confidence]

Nathan

From: Björn Grüning ***@***.***>
Date: Thursday, February 10, 2022 at 9:48 AM
To: BioContainers/multi-package-containers ***@***.***>
Cc: Nathan Spix ***@***.***>, Author ***@***.***>
Subject: Re: [BioContainers/multi-package-containers] Add biscuit-related tools (PR #2072)

It should be available, is it not?

—
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FBioContainers%2Fmulti-package-containers%2Fpull%2F2072%23issuecomment-1035007934&data=04%7C01%7C%7Ca91eb52c7f504bd54b4a08d9eca45878%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637801012867046579%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=6szJ%2FnS1S5Y66N0OSW0sfYp9ruzhLpYi8FXbXgeenG0%3D&reserved=0>, or unsubscribe<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FANSLFHUI2IM6XR5TB23WWCTU2PF2JANCNFSM5N6FGHRQ&data=04%7C01%7C%7Ca91eb52c7f504bd54b4a08d9eca45878%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637801012867046579%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Pt1aggVVfKNvbJEPGKMtnNVCqgdLeuKn2JFYTC7p6%2Bs%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7Ca91eb52c7f504bd54b4a08d9eca45878%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637801012867046579%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=bibyXX2bxlbmN3u1wmkmGZ6rqlZSK64NrzUM0UaaxKw%3D&reserved=0> or Android<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7Ca91eb52c7f504bd54b4a08d9eca45878%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637801012867046579%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=ESzAVQ2RiNysxj8tDN02R%2B1HZOpEodM0wZ1jTylqCyQ%3D&reserved=0>.

**Comment by njspix** [comment]
Hmm looking at the actions I do see the later (merge to master) action where it did upload. And it looks like I can successfully pull the image. Sorry for the confusion!

**Comment by bgruening** [comment]
Glad that it works.

**Pull request merged**